open-liberty/content.md

```diff
@@ -2,7 +2,9 @@
 
 The images in this repository contain Open Liberty. For more information about Open Liberty, see the [Open Liberty Website](https://openliberty.io/) site.
 
-If you're looking for Open Liberty based on Red Hat's Universal Base Image, please see [this repo](https://hub.docker.com/r/openliberty/open-liberty).
+This repository contains OpenLiberty based on top of OpenJDK 8 with Eclipse OpenJ9 JRE images only. See
```

Comment:
I would add " with Ubuntu", just so folks know it's OL + AdoptOpenJDK + Ubuntu  (since we also have OL + AdoptOpenJDK in the other repo, but with UBI instead)